tests/components/mqtt/test_init.py

```diff
@@ -790,6 +790,21 @@ def test_entity_device_info_schema() -> None:
             "configuration_url": "http://example.com",
         }
     )
+    # relative configuration_url
+    MQTT_ENTITY_DEVICE_INFO_SCHEMA(
+        {
+            "identifiers": ["helloworld", "hello"],
+            "connections": [
+                [dr.CONNECTION_NETWORK_MAC, "02:5b:26:a8:dc:12"],
+                [dr.CONNECTION_ZIGBEE, "zigbee_id"],
+            ],
+            "manufacturer": "Whatever",
+            "name": "Beer",
+            "model": "Glass",
+            "sw_version": "0.1-beta",
+            "configuration_url": "/api/hassio_ingress/XXXXXXXXXX",
```

Comment:
This is not needed, make urls like: `homeassistant://config/general`

Reply:
Thanks i did not find the information about the `homeassistant://` prefix.

I removed my modifications and added the following to tests/components/mqtt/test_init.py; tests do fail, is it expected that this prefix is not expanded in tests ?

```
+    # relative URL
+    MQTT_ENTITY_DEVICE_INFO_SCHEMA(
+        {
+            "identifiers": ["helloworld", "hello"],
+            "connections": [
+                [dr.CONNECTION_NETWORK_MAC, "02:5b:26:a8:dc:12"],
+                [dr.CONNECTION_ZIGBEE, "zigbee_id"],
+            ],
+            "manufacturer": "Whatever",
+            "name": "Beer",
+            "model": "Glass",
+            "sw_version": "0.1-beta",
+            "via_device": "test-hub",
+            "configuration_url": "homeassistant://api/hassio_ingress/XXXXXXX",
+        }
+    )
```

```
FAILED tests/components/mqtt/test_init.py::test_entity_device_info_schema - voluptuous.error.MultipleInvalid: invalid url for dictionary value @ data['configuration_url']

Results (6.03s):
     107 passed
       1 xpassed
       1 failed
         - tests/components/mqtt/test_init.py:753 test_entity_device_info_schema
```

Reply:
I think you might need this:
https://github.com/home-assistant/core/pull/96107